content/zh-cn/docs/reference/kubernetes-api/workload-resources/resource-claim-v1beta1.md

```diff
@@ -0,0 +1,1969 @@
+---
+api_metadata:
+  apiVersion: "resource.k8s.io/v1beta1"
+  import: "k8s.io/api/resource/v1beta1"
+  kind: "ResourceClaim"
+content_type: "api_reference"
+description: "ResourceClaim 描述对集群中供工作负载使用的资源的访问请求。"
+title: "ResourceClaim v1beta1"
+weight: 16
+---
+<!--
+api_metadata:
+  apiVersion: "resource.k8s.io/v1beta1"
+  import: "k8s.io/api/resource/v1beta1"
+  kind: "ResourceClaim"
+content_type: "api_reference"
+description: "ResourceClaim describes a request for access to resources in the cluster, for use by workloads."
+title: "ResourceClaim v1beta1"
+weight: 16
+auto_generated: true
+-->
+
+<!--
+The file was copied and updated manually from the v1alpha3 API.
+The content is not quite up-to-date, which needs to be fixed
+by generating the file automatically.
+-->
```

Comment:
Should we fix the English original first? Usually that's the workflow we follow.

Reply:
Seems yes. It's a relatively complicated task to [generate these reference](https://kubernetes.io/docs/contribute/generate-ref-docs/kubectl/). Need to wait for someone update them.